etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/MessageRequestHandler.java

```diff
@@ -12,10 +12,10 @@ public interface MessageRequestHandler<T> {
     /**
      * Parses the request, converts and sends the message
      *
-     * @param receivedSubmissionId the ID for the submission returned from ReportStream
```

Comment:
We could rephrase this for clarity. Something like: the report id created by ReportStream and sent to us in the request

Reply:
I used your suggestion.